BlockParty.py

```diff
@@ -464,9 +464,7 @@ def draw_score(self, cairo_ctx):
         displaystr += '\nLevel: ' + str(self.level)
         displaystr += '\nLines: ' + str(self.linecount)
 
-        cairo_ctx.set_source_rgb(self.color_score.red,
-                                 self.color_score.green,
-                                 self.color_score.blue)
+        cairo_ctx.set_source_rgb(0,0,0)
```

Comment:
Using all zeros for the colours to set for rgb isn't good for readability as not everyone will know at a first glance that the colour being set is black, you can use a `Gdk.RGBA` to parse whatever colour you need and use the values from that.

Reply:
I agree.  Even if not, needs space after commas.

Reply:
Done it in 163e3a5. Used the already used style to pass black color.